delete.md

```diff
@@ -0,0 +1,13 @@
+To clarify the current high-level differences with the artifact-manifest and the existing image-manifest:
+
+| Image-spec | Artifacts Spec |
+|-|-|
+| `config` REQUIRED | `config` optional as it's just another entry in the `blobs` collection with a `config mediaType` |
+| `layers` REQUIRED | `blobs`, which renamed `layers` to reflect general usage are OPTIONAL |
+| `layers` ORDINAL | `blobs` are defined by the specific artifact spec. Helm isn't ordinal, while other artifact types, like container images MAY make them ordinal |
+| `manifest.config.mediaType` used to uniquely identify different artifact types. | `manifest.artifactType` added to lift the workaround for using `manifest.config.mediaType` on a REQUIRED, but not always used property. |
+| | `subjectManifest` OPTIONAL, enabling an artifact to extend another artifact (SBOM, Signatures, Nydus, Scan Results, )
```

Comment:
Is there any concern in having subjectManifest REQUIRED? In the case it extends another artifact like another SBOM etc, it will include the manifest descriptor of the corresponding artifact

Reply:
Please see the current version at https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest-spec.md
subjectManifest is now optional.

Reply:
This link is a 404 for me. https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md seems to be the new location

Reply:
Ughh, github is good with repo renames, but not so good with file renames. We merged https://github.com/oras-project/artifacts-spec/pull/5 to cleanup filenames.

I fixed the link above.